colorcalculator.php

```diff
@@ -0,0 +1,74 @@
+<html>
+	<body>
+		<h1> Color Calculator </h1> <br> <br>
+			<form action=""
+
+				<p>First Color
+						<select name="color1" class="color1">
+							<option value='0'>--Select function--</option>
+							<option value='Red'> Red </option>
+							<option value='Yellow'> Yellow </option>
+							<option value='Blue'> Blue </option>
+						</select>
+				</p>
+				<br>
+				<p> mixed with </p>
+				<br>
+				<p>Second Color
+						<select name="color2" class="color2">
+							<option value='0'>--Select function--</option>
+							<option value='Red'> Red </option>
+							<option value='Yellow'> Yellow </option>
+							<option value='Blue'> Blue </option>
+						</select>
+				</p>
+				<input type= "submit"/> <br>
+
+			</form>
+	</body>
+</html>
+
+<?php
+
+//all error messages
+if (isset($_REQUEST['color1']) && isset($_REQUEST['color2']) && $_REQUEST['color1']=='0') {
```

Comment:
Nice job figuring this out. We're going to go over form validation soon, so if this seems inelegant, you're right, but it's the best way to do it for now.